Resources/views/Block/block_stats_base.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+{% extends sonata_block.templates.block_base %}
+
+{% block block %}
```

Comment:
This black names comes from the block bundle: https://github.com/sonata-project/SonataBlockBundle/blob/3.x/Resources/views/Block/block_base.html.twig#L12